inlang/source-code/paraglide/paraglide-js/src/cli/commands/init.ts

```diff
@@ -442,3 +471,36 @@ const prompt: typeof consola.prompt = async (message, options) => {
 	}
 	return response
 }
+
+export async function fileExists(filePath: string, nodeishFs: NodeishFilesystem) {
+	try {
+		await nodeishFs.access(filePath)
```

Comment:
not familar with access. the common pattern to do exeists checks is do a fs.stat and catch on throwing for non existing. access is not available on memfs.

Reply:
I switched it to the pattern you described so that it can work with lix's memfs